temporian/implementation/numpy/operators/arithmetic/base.py

```diff
@@ -13,24 +13,29 @@
 # limitations under the License.
 
 from typing import Dict
+from abc import ABC, abstractmethod
 
-from temporian.core.operators.arithmetic import ArithmeticOperation
-from temporian.core.operators.arithmetic import ArithmeticOperator
 from temporian.core.operators.arithmetic import Resolution
+from temporian.core.operators.arithmetic.base import BaseArithmeticOperator
 from temporian.implementation.numpy.data.event import NumpyEvent
 from temporian.implementation.numpy.data.event import NumpyFeature
-from temporian.implementation.numpy import implementation_lib
 from temporian.implementation.numpy.operators.base import OperatorImplementation
 
 
-class ArithmeticNumpyImplementation(OperatorImplementation):
-    def __init__(self, operator: ArithmeticOperator) -> None:
+class BaseArithmeticNumpyImplementation(OperatorImplementation, ABC):
```

Comment:
I think `ABC` parent class isn't needed here since `OperatorImplementation` has it already

Reply:
You're right, nice catch!